tools/metanovo/metanovo.xml

```diff
@@ -0,0 +1,336 @@
+<tool id="metanovo" name="MetaNovo" version="@TOOL_VERSION@+galaxy0" profile="20.09">
+  <description>
+    Produce targeted databases for mass spectrometry analysis.
+  </description>
+  <requirements>
+    <requirement type="package" version="@TOOL_VERSION@">metanovo</requirement>
+  </requirements>
+  <macros>
+    <token name="@TOOL_VERSION@">1.9.4</token>
+    <token name="@VERSION_SUFFIX@">0</token>
+    <import>macros_modifications.xml</import>
+  </macros>
+  <command>
+    <![CDATA[
+      #set $mgf_dir = 'mgf_files'
+      mkdir $mgf_dir &&
+      ln -s $input_fasta in.fasta &&
+
+      #if $input_type.type == "collection"
+      #for $input in $input_type.input_mgf_collection:
+      ln -s $input $mgf_dir/$input.element_identifier &&
```

Comment:
single quotes: ln -s '$input' '$mgf_dir/$input.element_identifier'

Reply:
Thanks @jj-umn. I've added single quotes for user input, and problem characters in the filenames are now replaced.